docs/guide/site_factory.md

```diff
@@ -0,0 +1,161 @@
+# Site Factory
+
+Site Factory is site management User Interface, integrated with Admin UI.
+ It allows you to configure sites without editing:
+
+- SiteAccess configuration
+- multisite configuration
+
+After [clean installation](../getting_started/install_ez_enterprise.md) the Site Factory will be disabled.
+This results in the following message on the **Site** tab:
+"There is a design configuration error, and you will not be able to create a new site. Please update the configuration."
+If you plan to use Site Factory you need to enable and configure it.
+
+To enable or disable Site Factory, follow respectively:
+
+- [Enable Site Factory section](#enable-site-factory)
+- [Disable Site Factory section](#disable-site-factory)
+
+## Enable Site Factory
+
+To enable Site Factory you need to set `enabled` to `true` in `vendor/ezsystems/ezplatform-site-factory/src/bundle/Resources/config/settings.yaml`.
+
+### Configure designs
+
+Next, configure Site Factory by adding empty SiteAccess groups in `config/packages/ezplatform.yaml`:
+
+```yaml
+ezplatform:
+    siteaccess:
+        list: [site, pl, de]
+        groups:
+            site_group: [site, pl, de]
+            sf_group_1: []
```

Comment:
maybe `site_factory_group_1` or `site_factory_group_A` or even `example_site_factory_group`

Reply:
only one empty group is mandatory